admin/class-admin.php

```diff
@@ -89,8 +89,8 @@ public function __construct() {
 		$this->check_php_version();
 		$this->initialize_cornerstone_content();
 
-		if ( Yoast_Network_Settings_API::meets_requirements() ) {
-			Yoast_Network_Settings_API::get()->register_hooks();
+		if ( Yoast_Network_Admin::meets_requirements() ) {
```

Comment:
I would move the `meets_requirements` check to the `register_hooks`. In this case you want to do: 
```
$network_admin = new Yoast_Network_Admin();
$network_admin->register_hooks();
```

In this case `register_hooks` has to find out to either set hooks or not.

Reply:
I made the check a static function so that you don't instantiate the class if the conditions are not met. Let me know what you think, maybe we can discuss on Slack if necessary.

I will add a call to it to the `register_hooks()` method, as an extra check to prevent these hooks from being added when they shouldn't.